.travis.yml

```diff
@@ -0,0 +1,32 @@
+branches:
+  only:
+  - trunk
+
+# Default settings for build stages
+language: java
+jdk:
+  - openjdk8
+  - oraclejdk8
+  - oraclejdk9
+
+install: true
```

Comment:
I think `install: true` does nothing?! You can probably leave it out, or does it serve a purpose? (like suppressing default behaviour of the `install` step)

Reply:
I guess I will remove it. Was just trying out different things.